spec/liquid_tags/tweet_tag_spec.rb

```diff
@@ -16,6 +16,13 @@ def generate_tweet_liquid_tag(id)
     end
   end
 
+  it "render properly", :vcr do
+    VCR.use_cassette("twitter_gem") do
+      liquid = generate_tweet_liquid_tag(twitter_id)
+      expect(liquid.render).not_to eq("liquid error: internal")
```

Comment:
It would be better to test for the expected working text, if the error message was to change at some point this test would no longer provide any assurances.

Reply:
You are right. Updating.